chatterbot/adapters/adaptation.py

```diff
@@ -0,0 +1,35 @@
+from chatterbot.utils.module_loading import import_module
+
+
+class Adaptation(object):
+    """
+    This is a base from which ChatterBot inherits utility methods
+    and a context attribute that allows access of adapters to be
+    shared between other adapters. This also makes it possible to
+    share other context information such as a name, or the current
+    conversation with each of the adapters.
+    """
+
+    class context(object):
```

Comment:
The main reason this is static is because it needs to hold the variable for the current conversation. Having it be static made more sense than passing the current conversation to each adapter every time a method on the adapter that needs that information is called.